torch/csrc/utils/throughput_benchmark.cpp

```diff
@@ -40,8 +40,7 @@ py::object ThroughputBenchmark::runOnce(py::args&& args, py::kwargs&& kwargs) {
 }
 
 ThroughputBenchmark::ThroughputBenchmark(jit::Module script_module)
-    : script_module_(script_module) {}
-
+    : script_module_(std::move(script_module)) {}
```

Comment:
jit module is a const ref type so I don't think this move is valid.

Reply:
For ref types it is possible to move them such as std::string_view. The problem here is the move constructors are implicitly defined as delete so that clang-tidy complains.